pkg/assembler/graphql/schema/isVulnerability.graphql

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines a GraphQL schema for the IsVulnerability. It contains a OSV, vulnerability that can be of type
+# cve or ghsa, justification, origin and collector
+"""
+IsVulnerability is an attestation that represents when an OSV ID represents a CVE or GHSA
+
+OSV - the osv object type that represents OSV and its ID
+Vulnerability - union type that consists of cve or ghsa
+Justification - the reason why the osv ID represents the cve or ghsa
+Origin - where this attestation was generated from (based on which document)
+Collector - the GUAC collector that collected the document that generated this attestation
+"""
+type IsVulnerability {
+  osv: OSV!
+  vulnerability: CveGhsaObject!
+  justification: String!
+  origin: String!
+  collector: String!
+}
+
+"""
+IsVulnerabilitySpec allows filtering the list of IsVulnerability to return.
+Only CVE or GHSA can be specified at once.
+"""
+input IsVulnerabilitySpec {
+  osv: OSVSpec
+  cve: CVESpec
+  ghsa: GHSASpec
+  justification: String
+  origin: String
+  collector: String
+}
+
+"""
+CveGhsaObject is a union of CVE and GHSA.
+"""
+union CveGhsaObject = CVE | GHSA
```

Comment:
should we name this `VulnObject`?

Reply:
`certifyVuln` has another union type that consists of CVE, OSV, and GHSA. So this was done to keep them separate